content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,36 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.7 [2020-03-27]
+
+### Features
+
+- Add option to display dashboards in [light mode](/v2.0/visualize-data/dashboards/control-dashboard/#toggle-dark-mode-and-light-mode).
+- Add [shell `completion` commands](/v2.0/reference/cli/influx/completion/) to the `influx` CLI.
+  specified shell (`bash` or `zsh`).
+- Make all `pkg` resources unique by `metadata.name` field.
+- Ensure Telegraf configuration tokens aren't retrievable after creation. New tokens can be created after Telegraf has been setup.
+- [Delete bucket by name](/v2.0/organizations/buckets/delete-bucket/#delete-a-bucket-by-name) using the `influx` CLI.
+- Add module to write line data to specified url, org, and bucket.
+- Add foundation for pkger stacks, stateful package management.
```

Comment:
should "pkger stacks" be "pkg stack"?

Add foundation for package management (`influx pkg`), including stacks and stateful package management?

link to (/v2.0/reference/cli/influx/pkg/)?

Reply:
I'd just say:

> Added `pkg stack` command to provide stateful package management.

With a link to `/v2.0/reference/cli/influx/pkg/stack`